technical-vision.md

```diff
@@ -0,0 +1,9 @@
+# OpenSSF Technical Vision
+
+We envision a future where participants in the open source software ecosystem focus on delivering high quality products and services, with security handled proactively, automatically, and as a matter of course.
+- Developers can easily learn secure development practices, and are proactively guided by their tools to apply those practices and automatically informed when action is needed to prevent, remediate, or mitigate security issues.
+- Developers, auditors and regulators can create and easily distribute security policies that are enforced through tooling and automation, providing continuous assurance of the results.
+- Developers and researchers can identify security issues (including unintentional vulnerabilities and malicious software) and have this information swiftly flow backwards through the supply chain to someone who can rapidly address the issue.
+- All stakeholders can provide information and notifications about product defects, mitigations, quality, and supportability and have this information rapidly flow forward across the ecosystem system to all users, and users can rapidly update their software or implement mitigations as appropriate.
+
+The Open Source Security Foundation (OpenSSF) provides tools, services, training, infrastructure, and resources to achieve this vision.
```

Comment:
For the rest, I think the statement on 9 needs to come first before the bullert points or we reword, as otherwise the prior statements sound like current facts. 

>  Our vision at The Open Source Security Foundation (OpenSSF) is to provide tools, services, training, infrastructure, and resources to help:

> - Developers learn secure development practices, and are proactively guided by their tools to apply those practices and automatically informed when action is needed to prevent, remediate, or mitigate security issues.
> - Developers, auditors and regulators to create and easily distribute security policies that are enforced through tooling and automation, providing continuous assurance of the results.

Reply:
We want to keep the focus on the future we envision. The last sentence is a short nod to how OpenSSF specifically contributes to this future.  Another way to say this is that we are writing a vision statement rather than a mission statement.

Reply:
ack, I don't mind the order too much. It's more that when I read it,  it sounds like the vision is the current status.

Reply:
How about this?  I added a colon to the intro sentence to clarify that the subsequent bullets are part of the future we envision, rather than standalone items.

>'We envision a future where participants in the open source ecosystem use and share high quality software, with security handled proactively and as a matter of course:'

Between that and the title of the document as "OpenSSF Technical Vision" this should be clear? Open to other suggestions.

Reply:
LGTM!